doc/user_guide/interactions/parameters.rst

```diff
@@ -366,62 +369,168 @@ selection:
 
     brush = alt.selection_interval()
-    ...
-    color=alt.when(brush).then(...)
-    size=alt.when(brush, empty=False).then(...)
-    ...
+
+    # Default behavior (empty=True): When nothing is selected, all points are gray
+    color=alt.when(brush).then("Origin:N").otherwise(alt.value("lightgray"))
+
+    # Custom behavior (empty=False): When nothing is selected, points keep their category colors
+    color=alt.when(brush, empty=False).then(alt.value("steelblue")).otherwise("Origin:N")
+
+Setting Initial Values
+"""""""""""""""""""""
+When creating interactive visualizations, you often want to guide users by
+starting with a pre-selected region rather than an empty selection. You can do this
+using the ``value`` parameter of :func:`selection_interval`.
+
+Setting initial values is particularly useful when:
+
+- Creating a filtered overview-detail pattern where a detailed subset is shown
+- Guiding viewer attention to a particular area of interest immediately
+- Preserving selection states across different visualizations or user sessions
+
+Here's an example using time series data where we initially focus on a specific date range
+containing the 2008 financial crisis:
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+    import datetime as dt
+
+    source = data.sp500.url
+
+    # Define initial date range to select
+    date_range = (dt.date(2007, 6, 30), dt.date(2009, 6, 30))
+
+    # Create interval selection with initial value
+    brush = alt.selection_interval(encodings=['x'],
+                                value={'x': date_range})
+
+    # Create base chart for both panels
+    base = alt.Chart(source, width=600, height=200).mark_area().encode(
+        x = 'date:T',
+        y = 'price:Q'
+    )
+
+    # Upper panel shows detailed view filtered by the brush
+    upper = base.encode(
+        alt.X('date:T').scale(domain=brush)
+    )
+
+    # Lower panel shows overview with the brush control
+    lower = base.properties(
+        height=60
+    ).add_params(brush)
+
+    # Combine the two charts
+    upper & lower
+
+This example demonstrates an "overview+detail" pattern - a common visualization technique
+where one chart shows the complete dataset while another shows a detailed view of the
+selected portion.
+
+When working with datetime values, you can use Python's native ``datetime.date`` or 
+``datetime.datetime`` objects directly. Altair automatically handles the conversion of 
+these objects to the appropriate format for Vega-Lite.
+
+The format of the ``value`` parameter depends on the encodings used in the selection:
+
+- For selections with `x` encoding: ``value={'x': [min, max]}``
+- For selections with `y` encoding: ``value={'y': [min, max]}``
+- For selections with both: ``value={'x': [xmin, xmax], 'y': [ymin, ymax]}``
+
+You can also use this feature with categorical data by specifying the categories to select. For example, to initially select only cars with 4 or 6 cylinders:
```

Comment:
```suggestion
You can also set initial values of a selection with categorical data. In this case the list/tuple specifies the exact categories to be selection rather than a range of categories. For example, to initially select only cars with 4 or 6 cylinders:
```

Since the example we use here selects based on a category that is a number, I think it is good to be explicit that it is not a range that is selected, as in the case for temporal data.